src/App.css

```diff
@@ -6,145 +6,240 @@
 }
 
 .mobile-nav-button {
-    display: none;
+  display: none;
 }
 
 .mob-nav-list-container {
-    position: absolute;
-    top: 30px; 
-    width: 100%;
-    text-align: center;
-    z-index: 1;
+  position: absolute;
+  top: 30px;
+  width: 100%;
+  text-align: center;
+  z-index: 1;
 }
 
 .mob-nav-list {
-    padding: 0;
-    background-color: #fff; /* Adjust the background color as needed */
-    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.5);
-    width: 60%;
-    margin-inline: auto;
+  padding: 0;
+  background-color: #fff; /* Adjust the background color as needed */
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.5);
+  width: 60%;
+  margin-inline: auto;
 }
 
 .mob-nav-list > li {
-    list-style: none;
-    padding-block: 15px;
+  list-style: none;
+  padding-block: 15px;
 }
 
 .sheSharp {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 24px;
-    font-weight: 500;
-    color: #2f4f4f;
-    display: flex;
-    justify-content: center;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 24px;
+  font-weight: 500;
+  color: #2f4f4f;
+  display: flex;
+  justify-content: center;
 }
 
 .styledData {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 16px;
-    line-height: 26px;
-    font-weight: 400;
-    color: #2f4f4f;
-    display: flex;
-    justify-content: center;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 16px;
+  line-height: 26px;
+  font-weight: 400;
+  color: #2f4f4f;
+  display: flex;
+  justify-content: center;
 }
 
 .container {
-    border-radius: 20px;
-    border-color: #2f4f4f;
-    border-style: solid;
-    border-width: 1px;
-    margin: auto;
-    width: 30%;
-    box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
-    padding: 40px;
-    transition: transform 0.3s ease-in-out;
+  border-radius: 20px;
+  border-color: #2f4f4f;
+  border-style: solid;
+  border-width: 1px;
+  margin: auto;
+  width: 30%;
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
+  padding: 40px;
+  transition: transform 0.3s ease-in-out;
 }
 
 .container:hover {
-    transform: scale(1.1);
+  transform: scale(1.1);
+}
+
+/* FOOTER STYLES */
+
+.footer {
+  background-color: #2f4f4f;
+  padding: 10px;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  color: #4fb39b;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.footer img {
+  width: 70px;
+}
+
+.logo_links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: 20px auto;
+}
+
+ul {
+  list-style-type: none;
+}
+
+ul li {
+  margin-bottom: 5px;
+}
+
+a {
+  text-decoration: none;
+  color: #4fb39b;
+}
+
+a:hover {
+  text-decoration: underline;
+  color: white;
+}
+
+.link-main-text {
+  color: white;
+  font-weight: bold;
+}
+/* .links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+} */
+.socials {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
I like the grid on the mobile view, but it feels quite bulky on the desktop view. Would it be possible for you to make them on one line for desktop, but to keep the grid for the media breakpoint?

Reply:
About this, the one-line layout didn't look too good on the official website, so I decided to do something different

Reply:
When you are ready with your changes, feel free to replace the screenshots or add new ones. And you can hit the little refresh icon so I know to check it again. Thank you!

Reply:
![Screenshot 2023-10-20 at 05 48 19](https://github.com/shesharpnl/hackathon-2023.react-js.base-example/assets/83243717/5459bc3c-bf48-4fc6-a396-f15963ca1b29)
![Screenshot 2023-10-20 at 05 48 09](https://github.com/shesharpnl/hackathon-2023.react-js.base-example/assets/83243717/e8827df6-c418-473c-8290-09e09a854699)
I've made the changes ma'am